src/test/java/org/fundacionjala/at15/katas/bankocr/vladimir/MostrarDatosImplTest.java

```diff
@@ -0,0 +1,296 @@
+package org.fundacionjala.at15.katas.bankocr.vladimir;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class MostrarDatosImplTest {
```

Comment:
Tests look good. It can be even better if responsabilities of this class could be delegated so implementation class and testing  class are not too big